docs/samples/metrics-and-monitoring/prometheus/clusterrolebinding.yaml

```diff
@@ -9,4 +9,4 @@ roleRef:
 subjects:
 - kind: ServiceAccount
   name: prometheus
-  namespace: kfserving-monitoring
\ No newline at end of file
+  namespace: default
```

Comment:
Maybe we can remove the namespace section and let kustomize to handle it? It is possible for kustomize v3 unless kustomize did some changes to the latest version.

e.g. https://github.com/kubeflow/manifests/blob/master/kfserving/kfserving-install/base/cluster-role-binding.yaml

Reply:
Yes. The latest commit removes namespace field altogether. It is still Kustomizable at the top-level so all good.